posttroll/__init__.py

```diff
@@ -69,5 +73,20 @@ def strp_isoformat(strg):
         return dat.replace(microsecond=mis)
 
 
+def _set_tcp_keepalive(socket):
+    keepalive = os.environ.get("POSTTROLL_TCP_KEEPALIVE")
```

Comment:
Could we use donfig to delegate the environment variable reading? https://github.com/pytroll/donfig

Reply:
I created #44 for this, there are other environment variables used in similar way.